FoodFriends/app/src/main/java/com/example/foodfriends/fragments/RestaurantDetailFragment.java

```diff
@@ -158,4 +214,39 @@ public void update(Observable o, Object arg) {
                     .into(ivToGo);
         }
     }
+
+    /**
+     * Adds all the restaurant's comments to the list
+     * */
+    private void queryComments() {
+        // specify what type of data we want to query - Post.class
+        ParseQuery<Comment> query = ParseQuery.getQuery(Comment.class);
+
+        query.whereEqualTo("restaurant", restaurant.getRestaurant());
```

Comment:
There could be a class that controls the commenting querying? Like CommentFetchHelper or some sort and would have methods that are responsible for saving/fetching comments?  

`public function queryForComments(restaurant, Callback/Listener for the return comments)`